src/metadata.json

```
@@ -2,7 +2,7 @@
    "uuid": "notification-position@drugo.dev",
    "name": "Notification Banner Position",
    "description": "Changes position of the notification banner from the default to the right side of the screen.",
-   "version": 4,
-   "shell-version": [ "3.36", "3.38", "40", "41", "42", "43" ],
+   "version": 5,
```

Can you make it `44`? I will match the version with the current supported Gnome version. 🤷🏼 
```suggestion
   "version": 44,
```

---

hehe I got rejected, you merged faster than I could change it :D 

Any how, thanks for the fast reply :)

---

Yeah, sorry... I just clicked the "accept suggestion" (my own, haha), and went ahead with merging.

I added minimal automation that should push to master the changes necessary when there's a new release of Gnome out there. I think I'll get a notification, which I'll use to push the artifact to the Gnome Extensions website.